metadata-ingestion/src/datahub/cli/migration_utils.py

```diff
@@ -20,44 +21,68 @@
 
 log = logging.getLogger(__name__)
 
+all_aspects = [
+    "schemaMetadata",
+    "datasetProperties",
+    "viewProperties",
+    "subTypes",
+    "editableDatasetProperties",
+    "ownership",
+    "datasetDeprecation",
+    "institutionalMemory",
+    "editableSchemaMetadata",
+    "globalTags",
+    "glossaryTerms",
+    "upstreamLineage",
+    "datasetUpstreamLineage",
+    "status",
+    "containerProperties",
+    "dataPlatformInstance",
+    "containerKey",
+    "container",
+    "domains",
+    "containerProperties" "editableContainerProperties",
+]
+
 
 def get_aspect_name_from_relationship_type_and_entity(
     relationship_type: str, entity_type: str
 ) -> str:
-    if relationship_type == "Produces":
-        if entity_type.lower() == "datajob":
-            return "dataJobInputOutput"
-    if relationship_type == "Consumes":
-        if entity_type.lower() == "chart":
-            return "chartInfo"
-        if entity_type.lower() == "datajob":
-            return "dataJobInputOutput"
-        if entity_type.lower() == "dataProcess":
-            return "dataProcessInfo"
-    if relationship_type == "DownstreamOf":
-        if entity_type.lower() == "dataset":
-            return "upstreamLineage"
-    if relationship_type == "ForeignKeyToDataset":
-        if entity_type.lower() == "dataset":
-            return "schemaMetadata"
-    if relationship_type == "DerivedFrom":
-        if entity_type.lower() == "mlfeature":
-            return "mlFeatureProperties"
-        if entity_type.lower() == "mlprimarykey":
-            return "mlPrimaryKeyProperties"
+    aspect_map = {
+        "Produces": {"datajob": "dataJobInputOutput"},
```

Comment:
This seems a bit difficult to maintain.

It also does not account for cases where the same edge type can be between multiple source dest nodes ... For examples "glossaryTerm ASSOCIATED_WITH dataset" and "tag ASSOCIATED_WITH dataset"... cc @gabe-lyons - maybe a reason not to do this?